src/error.rs

```diff
@@ -30,8 +30,7 @@ impl Error {
             Kind::Http(ref e) => Some(e),
             Kind::UrlEncoded(ref e) => Some(e),
             Kind::Json(ref e) => Some(e),
-            Kind::TooManyRedirects |
-            Kind::RedirectLoop => None,
+            Kind::TooManyRedirects | Kind::RedirectLoop => None,
```

Comment:
Likewise, rather keep the original styling here, and the other matches in this file.